### ISSUE TITLE
Dupe Guru not detecting duplicate files that have same MD5 hash

### ISSUE THREAD

**Issue by matttrv** [arsenetar/dupeguru#1232]
**Describe the bug**
Using DupeGuru for years coming to depend on its reliability.  Recently, ran it as a double check on some known duplicate files and was surprised when only 80-90% of the dupes were found.

I checked the hashes (MD5, as well as SHA-1) and they were definitely dupes.

I made some sample files to do a controlled test and experienced the same problem -- dupes are not identified.  I even loosened the criteria to search just on filename and no dupes found.

**To Reproduce**
1. Unzip the attached files they will extract into \banger\ and \hanger\ with a text file and a .url file in both folders.  I used .txt and .url files because they can be viewed in plain text so you know they are not malicious. (i'll attach the raw files as well)

2. Add the folders to DupeGuru as scan targets.  I scanned with both as "Normal" just to be the most inclusive about what could be identified as dupes.
![image](https://github.com/arsenetar/dupeguru/assets/77462293/e59421c2-d066-4765-82bb-ac294df6f133)

4. Check if dupes were found.  For me, none were found.
![image](https://github.com/arsenetar/dupeguru/assets/77462293/2febfb70-b18f-49f1-b2b5-0659e98bb97b)

6. Run a certutil (with MD5) against the files and see that the are indeed dupes.
![image](https://github.com/arsenetar/dupeguru/assets/77462293/a91a717c-224c-4193-a4a1-dfd66d4466a5)

**Expected behavior**
Should have been ID'd as dupes.  Tried on two different Windows 11 machines, same result, NOT identified as dupes.
Painful because if the dupes are not comprehensively identified, I'm not sure I can continue to use DupeGuru.
I guess it also makes me worry that things identified as dupes are not in fact dupes.

**Desktop (please complete the following information):**
 - OS: Windows 11
 - Version: DupeGuru 4.3.1   (tested back to 4.04, same issue)

![image](https://github.com/arsenetar/dupeguru/assets/77462293/84f66659-8a72-4d95-851c-2ad7fee1bde5)

[dupegurutest.zip](https://github.com/user-attachments/files/15601886/dupegurutest.zip)

[toby.txt](https://github.com/user-attachments/files/15602108/toby.txt)
[Alliance Stage Company.url.txt](https://github.com/user-attachments/files/15602475/Alliance.Stage.Company.url.txt)

EDIT: Attached the raw files as:
- toby.txt
- Alliance.Stage.Company.url.txt (dot TXT added because GitHub will not allow a dot URL to be uploaded)

**Comment by efanibi25** [comment]
Did you remove the size limit?

![image](https://github.com/arsenetar/dupeguru/assets/58971180/302b4476-4336-4a23-967b-9aea4f000981)

**Comment by AndroYD84** [comment]
![Screenshot 2024-06-21 093128](https://github.com/arsenetar/dupeguru/assets/58201286/0f764369-5e4c-41e9-b50b-112418cc71e2)
I'm pretty confident you forgot to change this default option.

**Comment by efanibi25** [comment]
> ![Screenshot 2024-06-21 093128](https://private-user-images.githubusercontent.com/58201286/341608527-0f764369-5e4c-41e9-b50b-112418cc71e2.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTk0OTk0OTksIm5iZiI6MTcxOTQ5OTE5OSwicGF0aCI6Ii81ODIwMTI4Ni8zNDE2MDg1MjctMGY3NjQzNjktNWU0Yy00MWU5LWI1MGItMTEyNDE4Y2M3MWUyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjclMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjI3VDE0Mzk1OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTA3ZWNkMzk1OWE4NDQ4MWJhOTJjNjRjNDg0YzlkM2IxNmVjZDY2NmVkNDc5OTMxNmM4ZGRiN2QxNDRiY2M1NDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.UPUEDrPVvSKrxiHL-H73ULKi34IcNECPSmlP1azGW-o) I'm pretty confident you forgot to change this default option.

Yes I was able to find the dupe files with that setting on, and  with the files given

**Comment by Dobatymo** [comment]
@matttrv @efanibi25 this can be closed then right?

**Comment by efanibi25** [comment]
> @efanibi25 this can be closed then right?

The op has never responded, but  the issue was fixed with the setting change. So I would closed it

@Dobatymo 
I can't close it, someone with permission will have to
